File: backend/albergue1/models/raza.py
from utils.db import db

class Raza(db.Model):
    __tablename__ = 'raza'

    # Variables
    id_raza = db.Column(
        db.Integer, 
        primary_key=True,
        autoincrement=True,
        nullable=False
    )

    nombre = db.Column(
        db.String(20),
        nullable=False,
        unique=True
    )

    # Constructor
    def __init__(self,
                nombre):
        self.nombre=nombre

File: frontend/src/app/models/raza.ts
export class Raza {
  id_raza : number;
  nombre: string;
  origen: string;

  constructor(
    id_raza: number,
    nombre: string,
    origen: string
  ) {
    this.id_raza = id_raza;
    this.nombre = nombre;
    this.origen = origen;
  }
}
